libminifi/include/utils/file/FileUtils.h

```diff
@@ -126,15 +126,24 @@ inline std::optional<std::filesystem::file_time_type> last_write_time(const std:
   return std::nullopt;
 }
 
-inline std::optional<std::string> format_time(const std::filesystem::file_time_type& time, const std::string& format) {
-  auto last_write_time_t = to_time_t(time);
+inline std::optional<std::string> format_time(const time_t& time, const std::string& format) {
   std::array<char, 128U> result{};
-  if (std::strftime(result.data(), result.size(), format.c_str(), gmtime(&last_write_time_t)) != 0) {
+  if (std::strftime(result.data(), result.size(), format.c_str(), gmtime(&time)) != 0) {
     return std::string(result.data());
   }
   return std::nullopt;
 }
 
+inline std::optional<std::string> format_time(const std::chrono::system_clock::time_point& time, const std::string& format) {
```

Comment:
We have some time_point to str utilities in TimeUtil.h, that deal with these problem. (maybe we should change the input parameter from sys_seconds to std::chrono::system_clock::timepoint)

Reply:
Makes sense, I've changed it in 9dc26a64f32870b99c9e9a562b37500b4638c943.